switch/oniguruma/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: uyjulian <uyjulian@gmail.com>
+
+pkgname=switch-oniguruma
+pkgver=6.9.2
+pkgrel=1
+pkgdesc='Oniguruma is a modern and flexible regular expressions library. It encompasses features from different regular expression implementations that traditionally exist in different languages.'
+arch=('any')
+url='https://github.com/kkos/oniguruma/'
+license=('BSD')
+options=(!strip libtool staticlibs)
+makedepends=('devkitpro-pkgbuild-helpers')
+source=("https://github.com/kkos/oniguruma/releases/download/v$pkgver/onig-$pkgver.tar.gz")
+sha256sums=('db7addb196ecb34e9f38d8f9c97b29a3e962c0e17ea9636127b3e3c42f24976a')
+
+build() {
+  cd onig-$pkgver
+
+  source /opt/devkitpro/devkita64.sh
+  source /opt/devkitpro/switchvars.sh
+
+  CFLAGS="${CFLAGS} -D__SWITCH__ " \
```

Comment:
We add -D__SWITCH__ to CPPFLAGS here https://github.com/devkitPro/pacman-packages/blob/master/pkgbuild-scripts/switchvars.sh#L9 if this library doesn't recognise CPPFLAGS then add CPPFLAGS to CFLAGS here.